### DISCUSSION TITLE
Ver 8.0. Plan

### DISCUSSION THREAD

**Discussion by black7375** [black7375/Firefox-UI-Fix#555] [Ideas]
## Concept: Colors

Motivation: We should respond to the theme of Firefox, which may have been scheduled.

- Current Dark/Light theme
  - https://bugzilla.mozilla.org/show_bug.cgi?id=1806945

![image](https://user-images.githubusercontent.com/25581533/210135998-ac18b3d0-bdf2-4fb6-b6f2-25547902612d.png)

- Windows 10 Native Menu
  - #545
  - [M1ch431/FirefoxW10ContextMenus](https://github.com/M1ch431/FirefoxW10ContextMenus)
- Windows 11 Color (Sun Valley)
  - [Guerra24/Firefox UWP Style](https://github.com/Guerra24/Firefox-UWP-Style)
- Private Window Color
  - Dark / Purple
- Transparent Tabbar
  - Windows: [MicaForEveryone](https://github.com/MicaForEveryone/MicaForEveryone)
  - Mac: [te6-in/lepton-custom](https://github.com/te6-in/lepton-custom)
  - Linux: [Filip-Sutkowy/blurclean-firefox-theme](https://github.com/Filip-Sutkowy/blurclean-firefox-theme)
- Linux's Non Native Menu like windows10
- Separate System Default Color/Layout
  - #356
- Long label moving at hover
  - #551

**Comment by 117649** [comment]
Can we have transparency/acrylic/mica context menu and application menu?
You can enable these on edge now and it's gorgeous.

**Reply by Aleksandra0056** [reply]
The transparent menus feature request is filed here: https://github.com/black7375/Firefox-UI-Fix/issues/585
